src/main/java/com/google/collegeplanner/data/Meeting.java

```diff
@@ -0,0 +1,122 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.package com.google.collegeplanner.data;
+
+package com.google.collegeplanner.data;
+
+import java.text.ParseException;
+import java.time.DayOfWeek;
+import java.util.ArrayList;
+import java.util.Collections;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+public class Meeting {
+  private ArrayList<DayOfWeek> days;
+  private String room;
+  private String building;
+  private int startTime;
+  private int endTime;
+
+  public Meeting(String days, String room, String building, 
+  int startTime, int endTime) throws ParseException {
+    this.days = new ArrayList<DayOfWeek>();
+    this.room = room;
+    this.building = building;
+    this.startTime = startTime;
+    this.endTime = endTime;
+
+    if (days.toUpperCase().contains("M")) {
+      this.days.add(DayOfWeek.MONDAY);
+    }
+    if (days.toUpperCase().contains("TU")) {
+      this.days.add(DayOfWeek.TUESDAY);
+    }
+    if (days.toUpperCase().contains("W")) {
+      this.days.add(DayOfWeek.WEDNESDAY);
+    }
+    if (days.toUpperCase().contains("TH")) {
+      this.days.add(DayofWeek.THURSDAY);
+    }
+    if (days.toUpperCase().contains("F")) {
+      this.days.add(DayOfWeek.FRIDAY);
+    }
+    if (days.toUpperCase().contains("S") && !days.toUpperCase().contains("SD")) {
```

Comment:
it's not immediately obvious what "SD" is - perhaps leave a comment for this section?

Reply:
Also my js code uses "Su" and "Sa". Don't know if we need to be consistent here or not.

Reply:
SD is to make sure I don't include tuesday, wednesday, and thursday as they all contain S but in the sequence "sday" while saturday and sunday do not, will just change the code to be consistent with Savion's js code